components/DblpImportModal.js

```diff
@@ -209,9 +209,11 @@ export default function DblpImportModal({ profileId, profileNames, updateDBLPUrl
         }, 2000)
       }
     } catch (error) {
-      setMessage(
-        'An error occurred while importing your publications. Please try again later.'
-      )
+      const errorMessage =
+        error.name === 'TooManyError'
+          ? 'DBLP import quota has reached'
```

Comment:
what happens if the user gets this message? ask them to try again?

Reply:
i think this does not apply any more because rate limiting has been replaced with maxReplies
let me remove it